src/common/serviceUtils.js

```diff
@@ -87,25 +86,32 @@ function handleApiMessages(messages) {
 export function handleRequestError(error) {
   // Validation errors
   if (error.response && error.response.data.field_errors) {
-    logInfo('Field Errors', error.response.data.field_errors);
+    logInfo('API field errors returned to client in field_errors field', { fieldErrors: error.response.data.field_errors });
     handleFieldErrors(error.response.data.field_errors);
   }
 
   // API errors
   if (error.response && error.response.data.errors !== undefined) {
-    logInfo('API Errors', error.response.data.errors);
+    logInfo('API errors returned to client in errors field', { errors: error.response.data.errors });
     handleApiErrors(error.response.data.errors);
   }
 
   // API messages
   if (error.response && error.response.data.messages !== undefined) {
-    logInfo('API Messages', error.response.data.messages);
+    logInfo('API errors returned to client in messages field', { messages: error.response.data.messages });
     handleApiMessages(error.response.data.messages);
   }
 
   // Single API error
   if (error.response && error.response.data.error_code) {
-    logInfo('API Error', error.response.data.errors);
+    logInfo('API error returned to client as response', {
+      errors: [
```

Comment:
1. Curious why this is an array?
2. We should loop back to this later.  `logAPIErrorResponse` has a lot of axios response parsing logic that should be factored out so it can be used with logInfo as well, and not have to be re-written.  See https://github.com/edx/frontend-logging/blob/master/src/NewRelicLoggingService.js#L50-L58

Reply:
For 1. - Not for a particularly good reason.  We were massaging a single error into an array elsewhere so I think I had it on the brain.
2. - Sounds great.  In working on this I felt like there were too many layers, so it'd be nice if all collapsed down a bit.

Reply:
I think I'm now more clear on these different cases.  I think, but correct me if I am wrong, that this might be leftover from a different/earlier api response design where `error_code` was put at the top-level of the response?  If you agree, can we use logging (potentially even logError) to help us confirm that this is never happening so we can loop back and delete this case altogether?